frontend/src/modules/contributor/components/details/contributor-details-work-history.vue

```diff
@@ -55,7 +57,7 @@
     </div>
 
     <lf-button
-      v-if="!masked && orgs.length > 3"
+      v-if="!masked && orgGrouped.length > minimumShownGroups"
       type="primary-link"
       size="medium"
       class="mt-6"
```

Comment:
This button is not aligned properly with each one of the experiences. It should have space between the experience and the button. I show it in the video

Reply:
Also, there is one behaviour which I'm not sure about the button position. Will ask Nuno